core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2597,6 +2605,23 @@ public void setAttribute(AlluxioURI path, SetAttributeOptions options)
   }
 
   /**
+   * Checks whether the owner belongs to the group.
+   *
+   * @param owner the owner to check
+   * @param group the group to check
+   * @throws FailedPreconditionException if owner does not belong to group
+   * @throws AccessControlException if this operation is not permitted
```

Comment:
under what circumstances would this method actually throw `AccessControlException`?